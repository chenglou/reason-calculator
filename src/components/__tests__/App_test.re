open Jest;
open Expect;

let _ =

describe "App" (fun _ => {
  test "renders" (fun _ => {
    let dispatch = fun _ => ();
    let state = Store.init;
    let tree = App.createElement ::state ::dispatch children::[] ()
      |> ReactShallowRenderer.renderWithRenderer;

    expect tree |> toMatchSnapshot;
  });
});
