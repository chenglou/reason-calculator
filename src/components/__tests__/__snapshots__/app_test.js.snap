// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<main
  className={
    Object {
      "data-css-hzy630": "",
    }
  }
>
  <Hero
    reasonProps={0}
  />
  <Calculator
    reasonProps={
      Array [
        Array [
          <Display
            reasonProps={
                  Array [
                        "0",
                      ]
            }
      />,
          Array [
            <Buttons
              reasonProps={
                      Array [
                              [Function],
                            ]
              }
      />,
            0,
          ],
        ],
      ]
    }
  >
    <Display
      reasonProps={
        Array [
          "0",
        ]
      }
    />
    <Buttons
      reasonProps={
        Array [
          [Function],
        ]
      }
    />
  </Calculator>
</main>
`;
